ee/modules/040-terraform-manager/images/terraform-manager-vcd/patches/fix-sizing-policy-changing.patch

```diff
@@ -0,0 +1,52 @@
+Subject: [PATCH] Power off the virtual machine if the sizing policy has been changed and the hot-add cpu or memory feature is disabled.
```

Comment:
Add readme.md with notes about this patch

Reply:
done